### ISSUE TITLE
Cannot decode incoming alerts from fink-broker: 

### ISSUE THREAD

**Issue by JulienPeloton** [astrolabsoftware/fink-client#26]
**Describe the issue**
Using the current version of fink-broker ([last commit](ca33ab27e5a3073dca88bf45fe2ceb7c7237c138)), the client cannot decode distributed alerts. Here are the steps to reproduce the error:

#### Step 1: Configure kafka cluster

Using `fink_kafka` from [fink-broker](https://github.com/astrolabsoftware/fink-broker), configure the cluster. Create a producer (for spark), and a consumer (for the client) with appropriate rights.

#### Step 2: deploy the broker

Launch fink broker (`stream2raw`, `raw2science`, `distribution`) and fire a stream.

#### Step 3: listen to the stream

Connect to the output stream with the client:

```
In [1]: from fink_client.consumer import AlertConsumer
   ...: topics = ["fink_cluster_rrlyr"]
   ...: config = {
   ...:   "bootstrap.servers": "************",
   ...:   "username": "************",
   ...:   "password": "***********",
   ...:   "group_id": "***********"
   ...: }

In [2]: consumer = AlertConsumer(topics, config)
Getting schema from fink servers...
```
#### Step 4: poll alerts

But then when polling alerts:

```
In [3]: topic, alert = consumer.poll(timeout=1)
---------------------------------------------------------------------------
UnicodeDecodeError                        Traceback (most recent call last)
<ipython-input-4-d5d6e0affb36> in <module>
----> 1 topic, alert = consumer.poll(timeout=ts)

~/Documents/workspace/myrepos/fink-client/fink_client/consumer.py in poll(self, timeout)
     92         topic = msg.topic()
     93         avro_alert = io.BytesIO(msg.value())
---> 94         alert = _decode_avro_alert(avro_alert, self._parsed_schema)
     95
     96         return topic, alert

~/Documents/workspace/myrepos/fink-client/fink_client/consumer.py in _decode_avro_alert(avro_alert, schema)
    234     """
    235     avro_alert.seek(0)
--> 236     return fastavro.schemaless_reader(avro_alert, schema)

fastavro/_read.pyx in fastavro._read.schemaless_reader()

fastavro/_read.pyx in fastavro._read.schemaless_reader()

fastavro/_read.pyx in fastavro._read._read_data()

fastavro/_read.pyx in fastavro._read.read_record()

fastavro/_read.pyx in fastavro._read._read_data()

fastavro/_read.pyx in fastavro._read.read_union()

fastavro/_read.pyx in fastavro._read._read_data()

fastavro/_read.pyx in fastavro._read.read_utf8()

fastavro/_six.pyx in fastavro._six.py3_btou()

UnicodeDecodeError: 'utf-8' codec can't decode byte 0xc2 in position 13: invalid continuation byte
```

In case this helps:

```
In [4]: msg = consumer._consumer.poll(1)

In [5]: msg.value()
Out[58]: b'\x00\x18ZTF17aabximu\x00\xbc\xf1\x8e\xcc\x89\xde\xd0\xda\x18\x00\nRRLyr\x00Q\xde\xc7Y\n\xc2BA\x00\x02\x00\x9e\xec\x9c\xc2\xe83\x00XQ\xa2A\x00rztf_20190610201400_000718_zg_c12_o_q1_scimrefdiffimg.fits\x00\x10Kulkarni\x00\x02\x00\xbc\xf1\x8e\xcc\x89\xde\xd0\xda\x18\x00\x02t\x00\x0c\x00\xf4\r\x00X\x00\x9c\x0b\x00\xfaZ\x1aE\x00]+\x12E\x00\x87\x03\xd7\xcb\xa5\xc0i@\x00\x88T\x07/\xb0\xd2D@\x00\xa9\x1d\x8aA\x00\xfbf\x87=\x007\xc3\xafA\x00w\ ....'
```

@cAbhi15, did you see such an error before? I thought you had made tests to connect the client to the broker directly?

**System Information:**
_Please update accordingly_
 - Operating System: `macOS-10.14.6`
 - Fink client version: `0.0.1`
 - Fink broker version: `0.2.1`
 - Occurred on which branch and with what commit: master, [last commit](ca33ab27e5a3073dca88bf45fe2ceb7c7237c138).

**Comment by cAbhi15** [comment]
@JulienPeloton
I did run tests connecting the client to fink broker directly. 

I didn’t see such an issue though.

Can I get back to you on this issue before eod ?

**Comment by JulienPeloton** [comment]
Thanks @cAbhi15 for your quick reply. I will keep investigating, and we can discuss later.

**Comment by JulienPeloton** [comment]
Additional information: the decoding works when both the broker and the client are in local mode (same machine). The error seen above arises when the broker is deployed remotely on the cloud and the client is installed in a local computer. Maybe library mismatch?

**Comment by JulienPeloton** [comment]
Fixed in #18  and https://github.com/astrolabsoftware/fink-broker/issues/298
